src/terms/ewald.jl

```diff
@@ -187,19 +186,24 @@ function energy_ewald(lattice, recip_lattice, charges, positions; η=nothing, fo
                     continue
                 end
 
-                dist = norm(lattice * (ti - tj - R))
+                v = lattice * (ti - tj - R)
```

Comment:
v is not such a great name... Δr  maybe?

Reply:
agreed!